<launch>
    <arg name="root_frame" default="base_link" />
    <arg name="target_frame" default="localized_object" />
    <arg name="arm_name" default="kinova" />

    <remap from="/joint_states" to="/$(arg arm_name)/joint_states" />
    <node pkg="blitzcrank" type="pregrasp_service" name="pregrasp_service" output="screen">
        <param name="root_frame" value="$(arg root_frame)" />
        <param name="target_frame" value="$(arg target_frame)" />
    </node>

</launch>
